quobyte_driver.go

```diff
@@ -72,16 +79,30 @@ func (driver quobyteDriver) Remove(request volume.Request) volume.Response {
 	return volume.Response{Err: ""}
 }
 
+func check_mount_point(mPoint string) error {
```

Comment:
Minor nitpick - can we make this camelCase, where the first letter is upper case iff the function should be exported?